### PR TITLE
New Port: Ghostris

### PR DESCRIPTION
**Pull request by Ganimoth**
# New Port for Ghostris

## Game Information
- **Title**: Ghostris
- **URL**: https://yaru.itch.io/ghostris

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [ ] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/ghostris/Ghostris.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+export PORT_32BIT="Y"
```
Comment (Cebion) [inline ports/ghostris/Ghostris.sh:17]: Is this a 32 bit port? If not needs to be removed

File: ports/ghostris/port.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "ghostris.zip",
+  "items": [
+    "Ghostris.sh",
+    "Ghostris"
```
Comment (Cebion) [inline ports/ghostris/port.json:6]: wrong case

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/ghostris/Ghostris.sh (hunk 1 above)
Comment (Ganimoth) [reply ports/ghostris/Ghostris.sh:17]: It is 32bit, but I can redo it in on gmloader if you want it easily distinguishable from the 64bit ones.

File: ports/ghostris/gameinfo.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<gameList>
+  <game>
+    <path>./Ghostris.sh</path>
+    <name>Ghostris</name>
+    <desc> Ghostris is a puzzle game where the blocks you place down are haunted and will do stuff behind your back as you try to clear lines. You need to think not just about how to match up blocks, but also how to deal with the blocks' movement patterns, and react to ghosts showing up to interfere with your puzzling. Can you survive the Ghostris?</desc>
+    <releasedate>20191129T000000</releasedate>
+    <developer>Yal</developer>
+    <publisher>Yal</publisher>
+    <genre>Puzzle</genre>
+    <image>./ghostris/screenshot.png</image>
```
Comment (cdeletre) [inline ports/ghostris/gameinfo.xml:11]: ghostrist-fs8
:)

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/ghostris/gameinfo.xml (hunk 3 above)
Comment (Ganimoth) [reply ports/ghostris/gameinfo.xml:11]: Thanks, I will add the cover :)

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**